charts/redpanda-console-oauth2proxy/templates/deployment.yaml

```diff
@@ -294,6 +294,7 @@ spec:
             - --reverse-proxy=true
             - --email-domain={{.Values.oauth2Proxy.emailDomain}}
             - --cookie-secure=true
+            - --auth-logging-format="{{.Client}} - {{.RequestID}} - [{{.Timestamp}}] [{{.Status}}] {{.Message}}"
```

Comment:
```suggestion
            # remove Username from audit log, to prevent PII in logs: https://oauth2-proxy.github.io/oauth2-proxy/configuration/overview/#auth-log-format
            - --auth-logging-format="{{.Client}} - {{.RequestID}} - [{{.Timestamp}}] [{{.Status}}] {{.Message}}"
```